.github/workflows/ci.yml

```
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    # By default, a workflow only runs when a pull_request 's activity type is
+    # opened , synchronize , or reopened. Adding ready_for_review here ensures
+    # that CI runs when a PR is marked as not a draft, since we skip CI when a
+    # PR is draft
```

Nitpick to avoid weird spacing:

```suggestion
    # By default, a workflow only runs when a pull_request's activity type is
    # opened, synchronize, or reopened. Adding ready_for_review here ensures
    # that CI runs when a PR is marked as not a draft, since we skip CI when a
    # PR is draft
```